showcase/package.json

```diff
@@ -34,82 +34,82 @@
     "test:a11y-report": "yarn build:packages && ENABLE_A11Y_AUDIT=true ENABLE_A11Y_MIDDLEWARE_REPORTER=true ember test"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.7",
-    "@ember/optional-features": "^2.1.0",
+    "@babel/core": "^7.26.0",
+    "@ember/optional-features": "^2.2.0",
     "@ember/string": "^3.1.1",
     "@ember/test-helpers": "^3.3.0",
     "@ember/test-waiters": "^3.1.0",
     "@embroider/test-setup": "^3.0.3",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@glint/core": "^1.4.0",
+    "@glint/core": "^1.5.0",
     "@glint/environment-ember-loose": "^1.4.0",
-    "@glint/environment-ember-template-imports": "^1.4.0",
-    "@glint/template": "^1.4.0",
+    "@glint/environment-ember-template-imports": "^1.5.0",
+    "@glint/template": "^1.5.0",
     "@hashicorp/design-system-components": "workspace:^",
     "@hashicorp/design-system-tokens": "workspace:^",
-    "@percy/cli": "^1.28.8",
+    "@percy/cli": "^1.30.5",
     "@percy/ember": "^4.2.0",
     "@tsconfig/ember": "^3.0.8",
-    "@types/qunit": "^2.19.10",
+    "@types/qunit": "^2.19.12",
     "@types/rsvp": "^4.0.9",
-    "@typescript-eslint/eslint-plugin": "^8.5.0",
-    "@typescript-eslint/parser": "^8.5.0",
+    "@typescript-eslint/eslint-plugin": "^8.18.1",
+    "@typescript-eslint/parser": "^8.18.1",
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^8.2.2",
     "ember-a11y-testing": "^6.1.1",
-    "ember-auto-import": "^2.7.3",
-    "ember-basic-dropdown": "^8.1.0",
+    "ember-auto-import": "^2.10.0",
+    "ember-basic-dropdown": "^8.4.0",
     "ember-body-class": "^3.0.0",
     "ember-cli": "~5.9.0",
     "ember-cli-babel": "^8.2.0",
     "ember-cli-clean-css": "^3.0.0",
-    "ember-cli-dependency-checker": "^3.3.2",
-    "ember-cli-deprecation-workflow": "^2.1.0",
+    "ember-cli-dependency-checker": "^3.3.3",
+    "ember-cli-deprecation-workflow": "^2.2.0",
     "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sass": "^11.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-string-helpers": "^6.1.0",
     "ember-cli-terser": "^4.0.2",
     "ember-concurrency": "^4.0.2",
-    "ember-deep-tracked": "^2.0.0",
+    "ember-deep-tracked": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-modifier": "^4.1.0",
     "ember-page-title": "^8.2.3",
-    "ember-power-select": "^8.2.0",
-    "ember-qunit": "^8.1.0",
-    "ember-resolver": "^11.0.1",
+    "ember-power-select": "^8.6.1",
```

Comment:
given [we're still on an earlier version in the components package](https://github.com/hashicorp/design-system/blob/a702f931291ecb0f3954b51a237a8d5d0ecef0bf/packages/components/package.json#L55), it may make sense to avoid updating this here and in the website for for now

Reply:
Reverted all upgrades to showcase and website `devDependencies` contained in the components `dependencies` this included `ember-a11y-refocus`, `ember-power-select`, and `sass`.